frontend/sw360-portlet/src/main/java/com/siemens/sw360/portal/users/SSOAutoLogin.java

```diff
@@ -60,8 +81,8 @@
             log.info(key + ":" + value);
```

Comment:
I dont think that this information should be logged (except for debugging purposes, maybe reduce logging level?)